noman.el

```diff
@@ -69,141 +70,115 @@
   (let ((subcommand (button-label button)))
     (noman (format "%s %s" noman--last-command subcommand))))
 
-(defun noman--exec (cmd suffix buffer)
-  "Execute CMD with the help SUFFIX and place the results in BUFFER."
-  (call-process "sh" nil buffer nil "-c" (concat cmd " " suffix)))
```

Comment:
`call-process` can be used directly.
This avoids the need to quote the  command's arguments and should be more performant.